README.md

```diff
@@ -128,9 +128,32 @@ wondering why your feature is not working.
 
 ### Test Helpers
 
-#### `withFeature`
+#### `enableFeature`
 
 Turns on a feature for the test in which it is called.
+Requires ember-cli-qunit >= 4.1.0. 
+
+Example:
+```js
+import { enableFeature } from 'ember-feature-flags/test-support';
+
+module('Acceptance | Awesome page', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('links go to the new homepage', async function (assert) {
+    enableFeature(this.owner, 'new-homepage');
+  
+    await visit('/');
+    await click('a.home');
+
+    assert.equal(currentRoute(), 'new.homepage', 'Should be on the new homepage');
+  });
+});
+```
+
+#### `withFeature`
+
+Older pre- RFC232 tests can utilize `withFeature` test helper to turn on a feature for the test.
```

Comment:
It was actually RFC268. But let's refer to them as "new" and "old" -style acceptance tests in the readme